CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+- Add callback to handle skipped files ([#864](https://github.com/getsentry/symbolic/pull/864))
```

Comment:
_nit_: You could mention what you added the callback to, or which skipped files this refers to. Symbolic is a large crate with many parts dealing with files, so this entry is a bit ambiguous.